### PR TITLE
feat: add autocomplete support

### PR DESCRIPTION
**Pull request by i4ki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,16 +2,18 @@
 
 # Table of Contents
 
+- [Table of Contents](#table-of-contents)
```
Comment (katcipis) [inline README.md]: hmm was this added with mdtoc ? it seems that it should not add this =P (thinking on removing Table Of Contents as a header in general, doesn't help much)

**Review by katcipis — APPROVED** [review]
:rocket:

File: README.md (hunk 1 above)
Comment (i4ki) [reply README.md]: no just reverted those changes... vscode f@#$% up the file when I saved it...

Comment (katcipis) [reply README.md]: ahh it seems to be some sort of automatic TOC generation on vscode too =P

**Pull request merged**